ritasamir/assignment1/Q_02.java

```diff
@@ -0,0 +1,28 @@
+package assignment_1;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+
+public class Q_02<T> {
+
+	/**
+	 * get the kth to last element of a linked list
+	 * @param linkedList
+	 * @param k
+	 * @return the element at position k from the back.
+	 */
+	public T getLastK(LinkedList<T> linkedList, int k) {

```

Comment:
Could you add a main method (or a unit test) for each of getLastK and checkAnagram and call them with a few examples, verifying that the result is correct?

Reply:
I created a JUnitTest and edited the checkAnagram method, Would you please take another look :smile_cat: ?